amplitude/config.go

```diff
@@ -50,3 +50,13 @@ func (c Config) IsValid() bool {
 func (c Config) IsMinIDLengthValid() bool {
 	return c.MinIDLength > 0
 }
+
+func (c Config) IsEmpty() bool {
+	return c.APIKey == "" && c.FlushInterval == 0 &&
```

Comment:
Is the same as `return c == Config{}`?

Reply:
It returns build failed error 
```
invalid operation: c == Config{} (struct containing func(event Event, code int, message ...string) cannot be compared)
```
As `c.Logger` is a function here

Reply:
Thanks, I haven't tried to compile. IDE doesn't show the error for some reason.